CopyKeys/CopyKeys.ps1

```diff
@@ -303,20 +309,31 @@ class Source
 class ConstantStrings
 {
     static [string] $adeExtensionPrefix = "azurediskencryption"
+    static [string] $apiVersion = "api-version"
     static [string] $asrSuffix = "-asr"
     static [string] $authHeader = "authorization"
     static [string] $contentType = "application/json"
+    static [string] $httpGet = "GET"
     static [string] $httpPost = "POST"
     static [int] $keyVaultNameMaxLength = 24
     static [string] $loadingBEK = "Loading target BEK vault"
     static [string] $loadingKEK = "Loading target KEK vault"
     static [string] $loadingRG = "Loading resource groups"
+    static [string] $managementAzureEndpoint = "https://management.azure.com"
```

Comment:
This azure endPoint does not work for fairfax region

Reply:
Using the arm endpoint using context.environment